src/feed_handler.py

```diff
@@ -322,5 +325,9 @@ def compare(self, other: StandardFeedHandler) -> bool:
             comparison_failed = self_book.compare_books(other_book)
             if comparison_failed:
                 failed = True
-        # TODO compare subaccounts
+
+        if self.get_subaccounts() != other.get_subaccounts():
```

Comment:
nit: can we get a couple more granular logs?
i.e subaccount length difference, also if X, Y,  matches but Z doesn't can we print out the differing values but not the matching ones?

Reply:
done